models/Activity/Award.ts

```diff
@@ -22,8 +23,8 @@ export interface AwardAssignment
   award: Award;
 }
 
-export class AwardModel extends Stream<Award>(ListModel) {
-  client = sessionStore.client;
+export class AwardModel extends Stream<Award>(StrapiListModel) {
```

Comment:
```suggestion
export class AwardModel extends StrapiListModel<Award> {
```
Strapi 有能定位到具体页数的分页参数，不需要 `Stream` 流式加载。

Reply:
看不懂上面的两次修改

Reply:
> 看不懂上面的两次修改

以前是附加流式加载特性，需要用 mixin（混入）的写法，也就是单继承语言实现多继承。

但 Strapi 不需要流式加载，就直接继承基类即可。